### DISCUSSION TITLE
Release App 3.1.0 & #1.0.3 (Build 7)

### DISCUSSION THREAD

**Discussion by sorydima** [sorydima/Marina.Moda-#30] [Announcements]
# Marina.Moda® 💖 Changelog

Revolutionize streaming with Marina.Moda® 💖♥️: a love-based blockchain and AI web service! 🐱‍🏍 In the ever-evolving world of streaming services, Marina.Moda® 💖♥️ is emerging as a unique and innovative platform that combines the power of blockchain and artificial intelligence (AI) to redefine the streaming experience. Also popularly known as Domestic Marina, this lovingly crafted web service promises a revolutionary approach to content delivery and user experience. **The Union of Blockchain and Streaming** Domestic Marina ♥️ stands out because it uses blockchain technology to solve some common problems that absolutely everyone faces!

![Снимок экрана от 2024-06-04 22-57-02](https://github.com/sorydima/Marina.Moda-/assets/43252008/dc4933d4-15c0-4950-8c93-7748d3d1fadb)

## Version 3.1.0

We're excited to announce the release of Marina.Moda® 3.1.0! This update brings a host of new features, enhancements, and fixes to make your experience even better.

### New Features
- **Synchronized Lyric Support**: Enjoy your favorite tracks with perfectly timed lyrics.
- **Improved Automatic Lyrics Importing**: Enhanced accuracy and speed for importing lyrics.
- **Infinite Scrolling on Search Page**: Load more matching results seamlessly as you scroll.
- **Mobile Context Menus**: Open context menus as trays for a smoother mobile experience.

### Fixes
- **og:image Tag**: Resolved issues with the og:image tag not displaying correctly.
- **Search Dropdown**: Prevented autofocus on the first item in the search dropdown.
- **Embed Images**: Fixed issues with track and album embed images not showing.
- **Search Ordering**: Corrected artist ordering issues in search results.
- **User Profile Page**: Non-public playlists will no longer appear on user profile pages.
- **SEO Tags**: Eliminated duplicate SEO tags.
- **Checkout Error Page**: Fixed incorrect links on the checkout error page.
- **API Docs Authentication**: Improved authentication process for API documentation.
- **Admin Area Settings**: Ensured settings are saved correctly in the admin area.
- **Sitemap Indexing**: Addressed issues with links to non-existing sitemaps.
- **Dark Mode Layout**: Fixed layout issues in the account settings page on dark mode.
- **Mobile Cookie Notice**: Improved layout for cookie notices on mobile devices.
- **Notification Marking**: "Mark all as unread" now affects all notifications, not just visible ones.

![photo_2024-06-04_22-58-36](https://github.com/sorydima/Marina.Moda-/assets/43252008/dcdcdfce-df56-49a5-ab9a-7df021027cbd)

### What's Changed
- **Dependency Updates**:
  - Bumped `share_plus` from 8.0.3 to 9.0.0 [#20](https://github.com/sorydima/Marina.Moda-/pull/20)
  - Bumped `flutter_lints` from 3.0.2 to 4.0.0 [#21](https://github.com/sorydima/Marina.Moda-/pull/21)
  - Updated `Constant.dart` [#22](https://github.com/sorydima/Marina.Moda-/pull/22)
  - Bumped `onesignal_flutter` from 3.5.3 to 5.2.0 [#29](https://github.com/sorydima/Marina.Moda-/pull/29)
  - Bumped `flutter_launcher_icons` from 0.9.3 to 0.11.0 [#28](https://github.com/sorydima/Marina.Moda-/pull/28)
  - Bumped `flutter_lints` from 1.0.4 to 2.0.3 [#27](https://github.com/sorydima/Marina.Moda-/pull/27)
  - Bumped `location` from 4.4.0 to 5.0.3 [#26](https://github.com/sorydima/Marina.Moda-/pull/26)
  - Bumped `google_mobile_ads` from 1.3.0 to 4.0.0 [#25](https://github.com/sorydima/Marina.Moda-/pull/25)

For a complete list of changes, please refer to the [full changelog](https://github.com/sorydima/Marina.Moda-/compare/3.0.9...3.1.0).

Thank you for your continued support and feedback. Happy updating! 🚀
https://rechain.ru/marinamodalog.html

Currently, this is a streaming platform with audio and video content, where anyone can become an artist, creator, or author. We also created a small social platform and network within the platform for communication and interaction. Now we are actively working on Blockchain solutions and the implementation of WEB3 solutions for the Metaverse of the Domestic Marina! ✌️

We have our own token based on the TON network - $MRN - Marina Token. We also released 2 NFT Art for public distribution, and we gave away one of them, and gave the second one randomly for a birthday! 🥳

https://tonviewer.com/UQCoeIyt8AQYwAwc11vWuSXZnedKgGPjZ9fkNlXwG56rO1Th/jetton/EQAiqK1qs1RUpDk2lyAV-1zs8P8zFPe3-ZA6gAHDxINY_fDn

We are a team of enthusiasts united by a common passion for music and innovation. We're building a unique community on TON ($TON) where everyone can enjoy socializing, sharing their favorite music, and playing exciting mini-games - all powered by our cutting-edge platform. ✨

![photo_2024-06-04_22-58-45](https://github.com/sorydima/Marina.Moda-/assets/43252008/e8ed42bb-5c5f-4e5b-a607-46270896cc1e)

We have - 🎧 Full access to music content: forget about paid subscriptions! We provide access to all your favorite tracks, albums and videos from platforms like Spotify, YouTube and SoundCloud - completely free and legal. ⭐️💬🐾

We are currently working on a music bot with mini-games: Our bot is your faithful companion in the world of music. It will not only help you find the track you want, but also play exciting mini-games that will make you love music even more. 😉

Join our community, give free rein to your musical passions, communicate with like-minded people, enjoy new discoveries and grow with us! 🎈

Welcome to the future of music social society! 🏌️

Our chat:
https://t.me/marinamodachat
https://matrix.to/#/#marinamoda:matrix.org

Our channels:
https://t.me/marinamodaru
https://t.me/marinamodaen

https://matrix.to/#/#marinamodaru:matrix.org
https://matrix.to/#/#marinamodaen:matrix.org

Platform 🌝:
https://marina.moda

All sorts of web3.0 stuff and our social networks:
https://adminmarina.online

#DomesticMarina #MarinaModa #TON #MRN

Join us in celebrating the launch of Marina.Moda Music City and immerse yourself in the sounds that define TON City. Stay tuned for more updates and exclusive content!

#MarinaModaMusicCity #TONCityStreaming #NewMusicExperience #ListenNow #ОтечественнаяМарина #MarinaModa #TON #MRN

---

For any inquiries or support, please contact our customer service at music@rechain.email

Happy Listening,
The Marina.Moda Music City Team

Copyright © Marina.Moda ® 💖♥️ 2023-2024, All Rights Reserved. Need help? 🤔 Email us! 👇 A Dmitry Sorokin production. All rights reserved. Powered by REChain. 🪐 Copyright © 2023-2024 REChain, Inc REChain ® is a registered trademark hr@rechain.email p2p@rechain.email pr@rechain.email sorydima@rechain.email support@rechain.email sip@rechain.email music@rechain.email cfa@rechain.email anti@rechain.email mot_cfa@rechain.email rechainstore@rechain.email models@rechain.email Please allow anywhere from 1 to 5 business days for E-mail responses! 💌 Our Stats! 👀 At the end of 2023, the number of downloads from the Open-Source Places, Apple AppStore, Google Play Market, and the REChain.Store ✨ exceeded 29 million downloads. 😈 👀

<hr /><em>This discussion was created from the release <a href='https://github.com/sorydima/Marina.Moda-/releases/tag/3.1.0'>Release App 3.1.0 & #1.0.3 (Build 7)</a>.</em>

**Comment by JollyNabeel** [comment]
Yes, you are doing a good job. I also have like the same [App Store ](https://sosomodpro.com/)for mooded apps and games. You can visit this app store and download your favorite apps and games.

**Comment by sorydima** [comment]
Build for the HUAWEI HARMONY OS with HMS CORE 

https://rechain.store/HUAWEI_MarinaModa.apk
